tests/incremental/hashes/function_interfaces.rs

```diff
@@ -104,17 +104,17 @@ pub fn order_of_parameters(p2: i64, p1: i32) {}
 // Unsafe ----------------------------------------------------------------------
 
 #[cfg(any(cfail1,cfail4))]
-pub fn make_unsafe() {}
+pub        fn make_unsafe() {}
```

Comment:
?

Reply:
This is to avoid spans from being changed in cfail5 and resulting in a difference in the optimized MIR.

Reply:
Huh, wild